package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "design-system",
-  "version": "0.14.4",
+  "version": "0.14.5”,
```

Comment:
This is causing build to fail:
```
npm ERR! Failed to parse json
npm ERR! Unexpected token '\n' at 3:24
npm ERR!   "version": "0.14.5â€�,
```
http://jenkins.wikia-prod:8080/job/design-system-reference-page-pr-check/705/console